doc/changelog/03-notations/11120-master+refactoring-application-printing.rst

```diff
@@ -0,0 +1,16 @@
+- **Fixed:** Inheritance of implicit arguments across notations made
+  uniform in parsing and printing. To the exception of notations of
```

Comment:
This first sentence sounds a bit convoluted to me. What about:

>Parsing and printing now handle inherited implicit arguments in notations the same.

? I'm not sure it fits exactly what you meant. In any case, we should try to aim for an active form and avoid the word "made".

Reply:
I made the following proposition: "Parsing and printing consistently handle inheritance of implicit arguments in notations". WDYT?